action.yml

```diff
@@ -17,14 +17,18 @@ inputs:
   destination_branch:
     description: Branch name to sync to
     required: true
-  ssh_private_key:
-    description: SSH key used to authenticate with git clone urls provided (optional if public or https clone url with authentication)
+  source_ssh_private_key:
+    description: SSH key used to authenticate with source ssh clone url provided (optional if public or https clone url with authentication)
```

Comment:
One thing to look into: since the default `id_rsa` is no longer used, we need to make sure public ssh clone urls still work without a `id_rsa`.